src/Utilities/functions.jl

```diff
@@ -184,22 +184,25 @@ end
 # Functions
 
 function map_indices(index_map::F, f::MOI.VectorOfVariables) where {F<:Function}
-    return MOI.VectorOfVariables(index_map.(f.variables))
```

Comment:
Broadcasting is more of a problem for latency than runtime. But if we can get away with a generator, it's probably better.

Reply:
Yes, I already noticed this. `MOI.VariableIndex[...]` is even faster but the different is not that big in my experience (at least in runtime, not in latency)